File: backend/src/main/java/com/NBE_4_5_2/Team5/global/Rq.java
package com.NBE_4_5_2.Team5.global;

import java.util.Optional;

public interface Rq {
	Optional<String> getRefreshToken();

	String getValueFromCookie(String accessToken);

	void addCookie(String accessToken, String s);

	String getHeader(String authorization);

	void removeCookie(String accessToken);
}

File: backend/src/main/java/com/NBE_4_5_2/Team5/global/rq/Rq.kt
package com.NBE_4_5_2.Team5.global.rq

import java.util.Optional

interface Rq {
    val refreshToken: Optional<String>

    fun getValueFromCookie(accessToken: String): String?

    fun addCookie(accessToken: String, s: String)

    fun getHeader(authorization: String): String?

    fun removeCookie(accessToken: String)
}
